corehq/apps/translations/integrations/transifex/client.py

```diff
@@ -209,10 +211,13 @@ def get_translation(self, resource_slug, hq_lang_code, lock_resource):
         return polib.pofile(temp_file.name)
 
     def get_project_langcodes(self):
-        languages = self._fetch_related(self.project, 'languages')
-        languages.append(self.project.source_language)
+        languages = self.get_all_project_languages()
```

Comment:
Intentionally excluded! I think this was breaking the backup function (the only place its used) because it kept complaining that the source language didn't exist on transifex. I realize I didn't investigate this super in depth but I'm pretty sure I could have the source language included in the backup file if need be. I assumed since all the pulled files have both the source language and target language columns it'd be redundant to have an excel sheet with just the source language columns (2 since it would treat it as both source and target).